WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/widget/utils/StatsIntentUtils.kt

```diff
@@ -29,4 +29,7 @@ private inline fun <reified T : Enum<T>> Intent.putEnumExtra(key: String, victim
 private inline fun <reified T : Enum<T>> Intent.getEnumExtra(key: String, default: T): T =
         getIntExtra(key, -1)
                 .takeUnless { it == -1 }
-                ?.let { T::class.java.enumConstants[it] } ?: default
+                ?.let {
+                    @Suppress("UNNECESSARY_SAFE_CALL")
```

Comment:
Is this `@Suppress` needed?

Reply:
Android studio complains when it's not there - I believe it's a false positive. 
<img width="814" alt="Screenshot 2020-09-23 at 11 37 40" src="https://user-images.githubusercontent.com/2261188/93995390-3c2eb280-fd91-11ea-9cb4-46818723382f.png">

Reply:
Interestingly Android Studio doesn't give me that error. 🤔

Certainly makes sense to include the `@Suppress` since you're seeing the warning though. 👍